Sources/Data/Model/Event.swift

```diff
@@ -35,3 +35,64 @@ public final class Event : NSManagedObject {
         self.timestamp = Date()
     }
 }
+
+extension Event {
+    /// Register a callback to be fired whenever Events are inserted.
+    ///
+    /// Note that this returns an opaque chit object that you must retain until you no longer wish the callback to be fired.
+    public static func observeNewEvents(
```

Comment:
This is now a bit awkward for a developer to use. Perhaps this makes more sense on the EventPipeline since we've decided we're keeping it?